metricbeat/module/elasticsearch/test_elasticsearch.py

```diff
@@ -27,10 +29,36 @@ def test_metricsets(self, metricset):
         """
         elasticsearch metricset tests
         """
+        if metricset == "ml_job":
+            self.create_ml_job()
+
         es = Elasticsearch(self.get_hosts())
         es.indices.create(index='test-index', ignore=400)
         self.check_metricset("elasticsearch", metricset, self.get_hosts(), self.FIELDS + ["service.name"])
 
     def get_hosts(self):
         return [os.getenv('ES_HOST', 'localhost') + ':' +
                 os.getenv('ES_PORT', '9200')]
+
+    def create_ml_job(self):
+        es = Elasticsearch(self.get_hosts())
+
+        # Enable xpack trial
+        try:
+            es.transport.perform_request('POST', "/_xpack/license/start_trial?acknowledge=true")
+        except:
+            print "Trial already enabled"
```

Comment:
We could print the error instead, in case it is caused by other reason.

Reply:
done